src/cmd/sql_cmd_test.cc

```diff
@@ -1474,6 +1474,7 @@ TEST_P(DBSDKTest, GlobalVariable) {
     auto cli = GetParam();
     cs = cli->cs;
     sr = cli->sr;
+    sr->Init();
```

Comment:
no need, did in `main` already: https://github.com/4paradigm/OpenMLDB/blob/d0c014210bf90d04b525202ff9c381c818d7f958/src/cmd/sql_cmd_test.cc#L1863

Reply:
but if we run all test, before this test, `session.execute_mode` will be set as `online`, so I init sr here to init session var and session var will as same as global var